entrypoint.sh

```diff
@@ -13,3 +13,8 @@ wget -N https://s3.amazonaws.com/Minecraft.Download/versions/${MINECRAFT_VERSION
 # Run the world renders (One pass to make map, one to generate points of interests)
 overviewer.py --config /home/minecraft/config.py
 overviewer.py --config /home/minecraft/config.py --genpoi
+
+# Add GMaps API key if API_KEY environment variable is set
+if [ "$API_KEY" ]; then
+  sed -i "s/https:\/\/maps.google.com\/maps\/api\/js/&?key=${API_KEY}/g" /home/minecraft/render/index.html
```

Comment:
Awesome work! I like this much better - it feels much more simple and easier to maintain. 

One piece of feedback - we can likely cut back on the escape characters by using a different [`sed` delimiters](http://backreference.org/2010/02/20/using-different-delimiters-in-sed/). For example, we could do something like:
```bash
sed -i "s|https://maps.google.com/maps/api/js/&?key=${API_KEY}|g"
```

Also, you're sure this works? I thought `sed` worked like this: `sed -i "s/search/replace/g"` and it seems like you're missing a segment there.

Reply:
Ah cool, I didn't know you could specify delimiters in `sed` :smile:. 

I used the `&` to reduce on the size of the command. Basically the `&` is a macro for "replace with the matched pattern". There is more information available [here.](http://www.grymoire.com/Unix/Sed.html#uh-3) If you want I change it to a more verbose version that includes url twice. Something like `sed "s|https://maps.google.com/maps/api/js|https://maps.google.com/maps/api/js?key=${API_KEY}|g" index.html`

I tested it out, running from a built container, and it worked. Below is a quick example.

```
$ cat index.html

<!DOCTYPE html>
<html>
<head>

<title>Minecraft Overviewer</title>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Minecraft-Overviewer 0.12.204 (22b9ab3)" />
<meta name="viewport" content="initial-scale=1.0, user-scalable=no" />

<link rel="stylesheet" href="overviewer.css" type="text/css" />

<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script type="text/javascript" src="https://maps.google.com/maps/api/js"></script>

<script type="text/javascript" src="underscore.js"></script>
<script type="text/javascript" src="backbone.js"></script>
<script type="text/javascript" src="overviewerConfig.js"></script>
<script type="text/javascript" src="overviewer.js"></script>
<script type="text/javascript" src="baseMarkers.js"></script>

</head>

<!-- Generated at: Wed, 13 Sep 2017 13:41:57 UTC -->
<body onload="overviewer.util.initialize()">
 <div id="mcmap">
     <div id="NoJSWarning" style="color:white; background-color:black">
         If you can see this message, there is likely a problem loading the Overviewer javascript components.
         Check the javascript console for error messages.
     </div>
 </div>
</body>
</html>

$ export API_KEY=TESTING
$ sed "s|https://maps.google.com/maps/api/js|&?key=${API_KEY}|g" index.html

<!DOCTYPE html>
<html>
<head>

<title>Minecraft Overviewer</title>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Minecraft-Overviewer 0.12.204 (22b9ab3)" />
<meta name="viewport" content="initial-scale=1.0, user-scalable=no" />

<link rel="stylesheet" href="overviewer.css" type="text/css" />

<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script type="text/javascript" src="https://maps.google.com/maps/api/js?key=TESTING"></script>

<script type="text/javascript" src="underscore.js"></script>
<script type="text/javascript" src="backbone.js"></script>
<script type="text/javascript" src="overviewerConfig.js"></script>
<script type="text/javascript" src="overviewer.js"></script>
<script type="text/javascript" src="baseMarkers.js"></script>

</head>

<!-- Generated at: Wed, 13 Sep 2017 13:41:57 UTC -->
<body onload="overviewer.util.initialize()">
 <div id="mcmap">
     <div id="NoJSWarning" style="color:white; background-color:black">
         If you can see this message, there is likely a problem loading the Overviewer javascript components.
         Check the javascript console for error messages.
     </div>
 </div>
</body>
</html>
```

Reply:
Nope, no need to change the `&`, that's sweet. I feel like I knew that, but I don't use it enough for it to look familiar. Thanks!